### PR TITLE
Fix crash on theme toggle while chat is open

### PR DESCRIPTION
**Pull request by gettinToasty**
We get into a bad state not properly awaiting chat reloading it's url when the theme changes which causes crashes on resize

### CODE REVIEW

File: app/services/chat.ts [hunk 1]
```diff
@@ -94,8 +94,9 @@ export class ChatService extends Service {
         nodeIntegration: false,
       },
     });
-
-    this.navigateToChat();
+    this.windowsService.updateStyleBlockers('main', true);
+    await this.navigateToChat();
+    this.windowsService.updateStyleBlockers('main', false);
```
Comment (andycreeth) [inline app/services/chat.ts]: What's the purpose of disabling style blockers here?  Is there a change this raises an exception and we get stuck with style blockers hidden?  Can this take a long time (servers time out after 60 seconds of loading) and we are stuck with style blockers hidden for that all that time?  This change makes me nervous.

Comment (gettinToasty) [reply app/services/chat.ts]: `navigateToChat` appears to catch errors from what i saw in the codebase, but the reason for this is rendering the browserWindow in chat while performing this navigation is what is causing the crashy state

Comment (andycreeth) [reply app/services/chat.ts]: It only catches 1 very specific type of error that happens when the page redirects, which is expected.  I think that style blockers is not the right thing to use here, and it needlessly hides many other things.  We should just hide the chat during this time.

Comment (gettinToasty) [reply app/services/chat.ts]: i asked if that was a viable fix in FE chat but got no responses 😅 will make that change

**Pull request merged**